Polyhedron/doc/Polyhedron/CGAL/Polyhedron_3.h

```diff
@@ -1118,6 +1150,13 @@ class Polyhedron_3 {
   */
   Facet_iterator facets_end();
 
+ /*!
+  returns a range of iterators over facets.
+  \note While the value type of `Facet_iterator` is `Vertex`, the value type of
+        `All_facet_handles::iterator` is `Facet_handle`
+  */
+  Facet_handles facet_handles();
```

Comment:
The documentation says `All_facet_handles` but the return type is `Facet_handles`.